src/main/java/driverutils/ChromeDriverManager.java

```diff
@@ -0,0 +1,28 @@
+package driverutils;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.BeforeClass;
+
+
+public class ChromeDriverManager implements Driverinterface {
+
+    public WebDriver setupChromeDriver() {
+        WebDriver driver = null;
+        if (driver == null) {
```

Comment:
unnecessary if condition - driver is always null here

Reply:
if condition deleted